src/main/java/bio/terra/cli/command/notebooks/Get.java

```diff
@@ -0,0 +1,40 @@
+package bio.terra.cli.command.notebooks;
+
+import bio.terra.cli.command.helperclasses.BaseCommand;
+import bio.terra.cli.command.helperclasses.options.Format;
+import bio.terra.cli.command.helperclasses.options.NotebookName;
+import bio.terra.cli.service.utils.GoogleAiNotebooks;
+import bio.terra.cloudres.google.notebooks.InstanceName;
+import com.google.api.services.notebooks.v1.model.Instance;
+import picocli.CommandLine;
+
+/** This class corresponds to the third-level "terra notebooks get-url" command. */
+@CommandLine.Command(
+    name = "get",
+    description = "Get the GCP data of an AI Notebook instance within your workspace.",
```

Comment:
For notebooks, it's not clear to me when I should use `resources describe` or `notebooks get`. I think the difference is that `resources describe` returns information that WSM knows about, and `notebooks get` returns information that the cloud knows about. If so, that seems confusing to users who aren't familiar with what metadata WSM does/not store. What do you think?

If we were already using CLI-specific objects (i.e. PF-729 were done), then it would make sense to me for this lookup code to live on the CLI Notebook Resource object. Since we are not doing that yet, (brainstorming) perhaps we can put this code in a utility method and call it from the `printText` function in the notebooks case block? With a todo to move it to the object once that ticket is ready.

Reply:
Agreed it's confusing. There's a split in our API/CLI about workspace operations around resources and resource-type specific operations. I don't know what the right way to resolve it is.

Calling GCP APIs from within a `printText` function seems kind of gross. (It also means we need more agruments for the workspace/global context to make the GCP call to get the notebook info.)
Ok if I just leave a TODO on `printText(ResourceDescription)` for now? If we get more feedback from the solutoins team, we can do more here.

Reply:
I'm hesitant to add a new command without a clear explanation of when to use it, though as with everything UX-related I would certainly defer to the solutions team if their feedback is different.

My preference is for the code to be ugly and the commands to look like we want, so that a future cleanup TODO is opaque to users. From the user's perspective, do you think this additional notebook information belongs in the output of:
1. `resources describe`: This was my first instinct. Agreed the code would be ugly for now, perhaps adding additional logic after `printText` in the `describe` command itself would be slightly less so?
2. `resources resolve`: Probably only relevant for the proxy url.
3. A separate `notebooks` command: 
4. rely on `gcloud`: This information is basically a shortcut for `gcloud notebooks instances describe`, with less-important properties filtered out right? So we could also leave this information out (with a TODO) until the refactoring is done, since users always have a way to look it up via `gcloud`.

More broadly, maybe this would be good to ask about at the next solutions team meeting -- if there's any larger preference around having useful Terra CLI wrappers for `gcloud` commands vs only using the Terra CLI for doing things via Terra services. I'm not sure what the right answer is, but I think the `notebooks start/stop/get` commands are the first examples we have of this.

Reply:
1. I'll go with that, maybe adding it in the `describe` command.
2. Yeah, only kind of makes sense for the proxy url.
3. I don't have a strong philosophical stance against this.
4. gcloud doesn't seem to be printing the proxy url for these service account proxy notebooks, so its insufficient.

One point for the Terra CLI resource type specific commands is that they let us use the workspace resource name to do the operation instead of only the GCP identifier. That means you don't have to translate the resolved name to the right flags for the gcloud command (`--project --location --instanceId` for notebooks.)

I'm strongly motivated to have `start`, `stop`, and someway to get the proxy URL/state in the CLI because it's just convenient for the manual testing/demoing I'm doing, so looking for some at least short term solution here. While we don't need to recreate every gcloud command available, I think it's nice to have some convenience for the most common ones. If we're strongly opposed to having any Terra CLI wrappers around something that's gcloud feasible, I'll remove `start` & `stop` but keep the uri  & state printing in `describe`.

Reply:
Agreed that using the resource name is a nice feature of wrapping them in Terra CLI commands.

RE: a separate `notebooks` command. If you decide this route, maybe a more disambiguated name would help (e.g. `terra notebooks status` for the state, `terra resolve --proxy-url`). That would eliminate confusion I think.